cookbooks/simple_ice_box.prm

```diff
@@ -0,0 +1,174 @@
+# A simple setup for using the boundary heat flux in a ice viscoplatic model.
+
+
+set Dimension                              = 2
+set Use years in output instead of seconds = true
+set End time                               = 1e6
+set Output directory                       = output-ice-box_2d
+set Nonlinear solver scheme                = single Advection, iterated Stokes 
+set Nonlinear solver tolerance             = 1e-5
+set Max nonlinear iterations               = 10
+set CFL number                             = 0.5
+set Pressure normalization                 = no
+set Use conduction timestep                = true
+# Material model
+# Rheology: Non-linear viscous flow and Drucker Prager Plasticity
+# Values for most rheological parameters are specified for a background material and
+# each compositional field.  Values for viscous deformation are based on diffusion creep 
+# based on Dombard and McKinnon 2006. 
+subsection Material model
+  set Model name = visco plastic
+
+  subsection Visco Plastic
+
+    # Reference temperature and viscosity
+    set Reference temperature = 273 
+    set Reference viscosity = 1e11
+
+    # The minimum strain-rate helps limit large viscosities values that arise
+    # as the strain-rate approaches zero.
+    # The reference strain-rate is used on the first non-linear iteration
+    # of the first time step when the velocity has not been determined yet.
+    set Minimum strain rate = 1.e-20
+    set Reference strain rate = 1.e-12
+
+    # Limit the viscosity with minimum and maximum values
+    set Minimum viscosity = 1e10
+    set Maximum viscosity = 1e22
```

Comment:
This is a fairly large range. Typically, we restrict the range to 6 or 8 orders of magnitude. Are there cases where the minimum and maximum values are actually present at a given point in time? If so, that speaks quite well to the robustness of the solvers 😄

Reply:
I need to figure out how to get the viscosity lower if I want there to be some overturn, at least that is what the analytical papers are suggesting.